lib/travis/build/appliances/maven_central_mirror.rb

```diff
@@ -0,0 +1,18 @@
+require 'travis/build/appliances/base'
+module Travis
+  module Build
+    module Appliances
+      class MavenCentralMirror < Base
+        GOOGLE_MAVEN_MIRROR='https://maven-central.storage-download.googleapis.com/repos/central/data/'
```

Comment:
can this please be configurable via config as we might not want to set a mirror by default

Reply:
especially for on prem customers

Reply:
sure. that's reasonable. we should first test this is a feasible solution to the problem.